### PR TITLE
Explicitly note Anaconda's TOS, mention miniforge more, update landing page.

### LINKED ISSUE #14225: Review docs for cases of inaccurate or outdated information related to agnostic channel management
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

As discussed in #14217, we should review the conda docs for cases where the conda CLI docs (https://docs.conda.io/projects/conda/) may contain outdated information regarding channel management.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by jezdez**
### Description

Closes #14225.
Refs #14217.

The landing page looks something like this now:

CleanShot 2024-09-24 at 13 20 07@2x

### Checklist - did you ...

- [ ] Add a file to the `news` directory ([using the template](https://github.com/conda/conda/blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

**Comment by jezdez** [comment]
@SylvainCorlay Any feedback on this?

File: docs/source/index.rst [hunk 1]
```diff
@@ -9,67 +9,94 @@ using conda in your own projects.
...
+We recommend the following conda distribtions to install conda:
 
-.. tab-set::
+.. grid:: 2
 
-    .. tab-item:: Windows :fab:`windows`
+    .. grid-item-card:: Miniconda
 
-        Miniconda installer for:
+        `Miniconda <https://docs.anaconda.com/miniconda>`_ is an installer
```
Comment (jaimergp) [inline docs/source/index.rst:18]:
```suggestion
        `Miniconda <https://docs.anaconda.com/miniconda>`__ is an installer
```

File: docs/source/index.rst [hunk 2]
```diff
@@ -9,67 +9,94 @@ using conda in your own projects.
...
+        by `Anaconda <https://anaconda.com/>`_ that comes
```
Comment (jaimergp) [inline docs/source/index.rst:19]:
```suggestion
        by `Anaconda <https://anaconda.com/>`__ that comes
```

File: docs/source/index.rst [hunk 3]
```diff
@@ -9,67 +9,94 @@ using conda in your own projects.
...
 
-                    macOS with Apple Silicon 64-bit :octicon:`download`
+            :fab:`linux` Linux :bdg-light-line:`aarch64 (arm64)` :octicon:`download`
 
-            .. grid-item::
+        ++++
 
-                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg
-                    :color: primary
+        Or with :fa:`beer-mug-empty` `Homebrew <https://brew.sh/>`_:
```
Comment (jaimergp) [inline docs/source/index.rst:50]:
```suggestion
        Or with :fa:`beer-mug-empty` `Homebrew <https://brew.sh/>`__:
```

File: docs/source/index.rst [hunk 4]
```diff
@@ -9,67 +9,94 @@ using conda in your own projects.
...
 
-                    macOS with Intel CPU 64-bit :octicon:`download`
+        .. code-block:: bash
+
+            brew install miniconda
+
+    .. grid-item-card:: Miniforge
+
+        Miniforge is an installer maintained by the `conda-forge community <https://
+        conda-forge.org>`_ that comes preconfigured for use with the conda-forge
```
Comment (jaimergp) [inline docs/source/index.rst:59]:
```suggestion
        conda-forge.org>`__ that comes preconfigured for use with the conda-forge
```

File: docs/source/index.rst [hunk 5]
```diff
@@ -9,67 +9,94 @@ using conda in your own projects.
...
 
-        Run the following `Homebrew <https://brew.sh/>`_ command:
+        .. button-link:: https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh
+            :color: primary
+
+            :fab:`linux` Linux :bdg-light-line:`aarch64 (arm64)` :octicon:`download`
+
+        +++
+
+        Or with :fa:`beer-mug-empty` `Homebrew <https://brew.sh/>`_:
```
Comment (jaimergp) [inline docs/source/index.rst:89]:
```suggestion
        Or with :fa:`beer-mug-empty` `Homebrew <https://brew.sh/>`__:
```

File: docs/source/index.rst [hunk 6]
```diff
@@ -9,67 +9,94 @@ using conda in your own projects.
...
 
-            brew install miniconda
+            brew install miniforge
+
 
 .. raw:: html
 
-    <p class="text-small">For more detailed instructions, see <a href="https://docs.anaconda.com/free/miniconda/">Miniconda's installation guide</a></p>
+    <p class="text-small">For more detailed instructions, see <a href="https://docs.anaconda.com/miniconda/">Miniconda's installation guide</a> and
+    <a href="https://conda-forge.org/download/">conda-forge's download site</a></p>
```
Comment (jaimergp) [inline docs/source/index.rst:99]:
```suggestion
    <p class="text-small">For more detailed instructions, see <a href="https://docs.anaconda.com/miniconda/" target="_blank">Miniconda's installation guide</a> and
    <a href="https://conda-forge.org/download/" target="_blank">conda-forge's download site</a></p>
```

File: docs/source/user-guide/configuration/mirroring.rst [hunk 7]
```diff
@@ -10,6 +10,10 @@ The default setup
 By default, ``conda`` can serve packages from two main locations:
 
 - ``repo.anaconda.com``: this is where ``defaults`` points to by default.
+  The default has been marked for pending deprecation as of conda 24.9.0,
```
Comment (jaimergp) [inline docs/source/user-guide/configuration/mirroring.rst:13]:
```suggestion
  The default value has been marked for pending deprecation as of conda 24.9.0,
```

File: docs/source/user-guide/getting-started.rst [hunk 8]
```diff
@@ -10,14 +10,29 @@ This guide to getting started with conda goes over the basics of starting up and
 
    Anaconda Navigator is a graphical desktop application that enables you to use conda without having to run commands at the command line.
 
-   See `Getting started with Anaconda Navigator <https://docs.anaconda.com/free/navigator/getting-started>`_ to learn more.
+   See `Getting started with Anaconda Navigator <https://docs.anaconda.com/navigator/getting-started/>`_ to learn more.
```
Comment (jaimergp) [inline docs/source/user-guide/getting-started.rst:13]:
```suggestion
   See `Getting started with Anaconda Navigator <https://docs.anaconda.com/navigator/getting-started/>`__ to learn more.
```

File: docs/source/user-guide/getting-started.rst [hunk 9]
```diff
@@ -10,14 +10,29 @@ This guide to getting started with conda goes over the basics of starting up and
...
 
 Before you start
 ================
 
-To bootstrap a conda installation, use a minimal installer such as `Miniconda <https://docs.anaconda.com/free/miniconda/>`_ or `Miniforge <https://conda-forge.org/download>`_.
+To bootstrap a conda installation, use a minimal installer such as `Miniconda <https://docs.anaconda.com/miniconda/>`_ or `Miniforge <https://conda-forge.org/download>`_.
 
-Conda is also included in the `Anaconda Distribution <https://docs.anaconda.com/free/anaconda/install/>`_.
+Conda is also included in the `Anaconda Distribution <https://docs.anaconda.com/anaconda/install/>`_.
```
Comment (jaimergp) [inline docs/source/user-guide/getting-started.rst:20]:
```suggestion
``conda`` is also included in the `Anaconda Distribution <https://docs.anaconda.com/anaconda/install/>`__.
```

File: docs/source/user-guide/getting-started.rst [hunk 10]
```diff
@@ -10,14 +10,29 @@ This guide to getting started with conda goes over the basics of starting up and
...
+
+.. note::
+
+    Miniconda and Anaconda Distribution come preconfigured to use the `Anaconda
+    Repository <https://repo.anaconda.com/>`_ and installing/using packages
```
Comment (jaimergp) [inline docs/source/user-guide/getting-started.rst:25]:
```suggestion
    Repository <https://repo.anaconda.com/>`__ and installing/using packages
```

File: docs/source/user-guide/getting-started.rst [hunk 11]
```diff
@@ -10,14 +10,29 @@ This guide to getting started with conda goes over the basics of starting up and
...
+    from that repository is governed by the `Anaconda Terms of Service
+    <https://www.anaconda.com/terms-of-service>`_, which means that it *might*
```
Comment (jaimergp) [inline docs/source/user-guide/getting-started.rst:27]:
```suggestion
    <https://www.anaconda.com/terms-of-service>`__, which means that it *might*
```

File: docs/source/user-guide/getting-started.rst [hunk 12]
```diff
@@ -10,14 +10,29 @@ This guide to getting started with conda goes over the basics of starting up and
...
+    require a commercial fee license. There are exceptions for individuals,
+    universities and companies with fewer than 200 employees (as of September
+    2024).
+
+    Please review the `terms of service <https://www.anaconda.com/terms-of-service>`_, Anaconda's most recent `Update on Anaconda’s Terms of Service for Academia
+    and Research <https://www.anaconda.com/blog/update-on-anacondas-terms-of-service-for-academia-and-research>`_,
+    and the `Anaconda Terms of Service FAQ
+    <https://www.anaconda.com/pricing/terms-of-service-faqs>`_ to answer your questions.
```
Comment (jaimergp) [inline docs/source/user-guide/getting-started.rst:35]:
```suggestion
    Please review the `terms of service <https://www.anaconda.com/terms-of-service>`__, Anaconda's most recent `Update on Anaconda’s Terms of Service for Academia
    and Research <https://www.anaconda.com/blog/update-on-anacondas-terms-of-service-for-academia-and-research>`__,
    and the `Anaconda Terms of Service FAQ
    <https://www.anaconda.com/pricing/terms-of-service-faqs>`__ to answer your questions.
```

File: docs/source/user-guide/install/index.rst [hunk 13]
```diff
@@ -7,7 +7,7 @@ The following are the most popular installers currently available:
 
 .. glossary::
 
-    `Miniconda <https://docs.anaconda.com/free/miniconda/>`_
+    `Miniconda <https://docs.anaconda.com/miniconda/>`_
```
Comment (jaimergp) [inline docs/source/user-guide/install/index.rst:10]:
```suggestion
    `Miniconda <https://docs.anaconda.com/miniconda/>`__
```

File: docs/source/user-guide/install/index.rst [hunk 14]
```diff
@@ -21,10 +21,22 @@ The following are the most popular installers currently available:
...
+
+.. note::
+
+    Miniconda and Anaconda Distribution come preconfigured to use the `Anaconda
+    Repository <https://repo.anaconda.com/>`_ and installing/using packages
+    from that repository is governed by the `Anaconda Terms of Service
+    <https://www.anaconda.com/terms-of-service>`_, which means that it *might*
+    require a commercial fee license. There are exceptions for individuals,
+    universities and companies with fewer than 200 employees (as of September
+    2024).
```
Comment (jaimergp) [inline docs/source/user-guide/install/index.rst:34]:
```suggestion
    Miniconda and Anaconda Distribution come preconfigured to use the `Anaconda
    Repository <https://repo.anaconda.com/>`__ and installing/using packages
    from that repository is governed by the `Anaconda Terms of Service
    <https://www.anaconda.com/terms-of-service>`__, which means that it *might*
    require a commercial fee license. There are exceptions for individuals,
    universities and companies with fewer than 200 employees (as of September
    2024).
```

File: docs/source/user-guide/install/index.rst [hunk 15]
```diff
@@ -75,8 +87,8 @@ Cryptographic hash verification
 ===============================
 
 SHA-256 checksums are available for
-`Miniconda <https://docs.anaconda.com/free/miniconda/>`_ and
-`Anaconda Distribution <https://docs.anaconda.com/free/anaconda/reference/hashes/all/>`_.
+`Miniconda <https://docs.anaconda.com/miniconda/>`_ and
+`Anaconda Distribution <https://docs.anaconda.com/anaconda/hashes/>`_.
```
Comment (jaimergp) [inline docs/source/user-guide/install/index.rst:91]:
```suggestion
`Miniconda <https://docs.anaconda.com/miniconda/>`__ and
`Anaconda Distribution <https://docs.anaconda.com/anaconda/hashes/>`__.
```

File: docs/source/user-guide/troubleshooting.rst [hunk 16]
```diff
@@ -404,7 +404,7 @@ Install using the ``--force`` option.
 
 Download and install the appropriate Miniconda
 for your operating system from the `Miniconda download page
-<https://docs.anaconda.com/free/miniconda/>`_ using the force option
+<https://docs.anaconda.com/miniconda/>`_ using the force option
```
Comment (jaimergp) [inline docs/source/user-guide/troubleshooting.rst:407]:
```suggestion
<https://docs.anaconda.com/miniconda/>`__ using the force option
```

File: docs/source/user-guide/tasks/manage-environments.rst [hunk 17]
```diff
@@ -379,7 +379,7 @@ not global.
    for "All Users", we add it to the *system* PATH. In the former case,
    you can end up with system PATH values taking precedence over
    your entries. In the latter case, you do not. *We do not recommend*
-   `multi-user installs <https://docs.anaconda.com/free/anaconda/install/multi-user/>`_.
+   `multi-user installs <https://docs.anaconda.com/anaconda/install/multi-user/>`_.
```
Comment (jaimergp) [inline docs/source/user-guide/tasks/manage-environments.rst:382]:
```suggestion
   `multi-user installs <https://docs.anaconda.com/anaconda/install/multi-user/>`__.
```

File: docs/source/user-guide/install/windows.rst [hunk 18]
```diff
@@ -5,7 +5,7 @@ Installing on Windows
 #. Download the installer:
 
    * `Miniconda installer for
-     Windows <https://docs.anaconda.com/free/miniconda/>`_
+     Windows <https://docs.anaconda.com/miniconda/>`_
```
Comment (jaimergp) [inline docs/source/user-guide/install/windows.rst:8]:
```suggestion
     Windows <https://docs.anaconda.com/miniconda/>`__
```

File: docs/source/user-guide/install/windows.rst [hunk 19]
```diff
@@ -5,7 +5,7 @@ Installing on Windows
...
 
    * `Anaconda Distribution installer for
      Windows <https://www.anaconda.com/download/>`_
```
Comment (jaimergp) [inline docs/source/user-guide/install/windows.rst:11]:
```suggestion
     Windows <https://www.anaconda.com/download/>`__
```

File: docs/source/user-guide/install/macos.rst [hunk 20]
```diff
@@ -10,9 +10,9 @@ Installing on macOS
 
 #. Download the installer:
 
-   * `Miniconda installer for macOS <https://docs.anaconda.com/free/miniconda/>`_.
+   * `Miniconda installer for macOS <https://docs.anaconda.com/miniconda/>`_.
```
Comment (jaimergp) [inline docs/source/user-guide/install/macos.rst:13]:
```suggestion
   * `Miniconda installer for macOS <https://docs.anaconda.com/miniconda/>`__.
```

File: docs/source/user-guide/install/macos.rst [hunk 21]
```diff
@@ -10,9 +10,9 @@ Installing on macOS
...
 
-   * `Anaconda installer for macOS <https://www.anaconda.com/download/>`_.
+   * `Anaconda Distribution installer for macOS <https://www.anaconda.com/download/>`_.
```
Comment (jaimergp) [inline docs/source/user-guide/install/macos.rst:15]:
```suggestion
   * `Anaconda Distribution installer for macOS <https://www.anaconda.com/download/>`__.
```

File: docs/source/user-guide/install/linux.rst [hunk 22]
```diff
@@ -4,7 +4,7 @@ Installing on Linux
 
 #. Download the installer:
 
-   * `Miniconda installer for Linux <https://docs.anaconda.com/free/miniconda/>`_.
+   * `Miniconda installer for Linux <https://docs.anaconda.com/miniconda/>`_.
```
Comment (jaimergp) [inline docs/source/user-guide/install/linux.rst:7]:
```suggestion
   * `Miniconda installer for Linux <https://docs.anaconda.com/miniconda/>`__.
```

File: docs/source/user-guide/install/linux.rst [hunk 23]
```diff
@@ -4,7 +4,7 @@ Installing on Linux
...
 
    * `Anaconda Distribution installer for Linux <https://www.anaconda.com/download/>`_.
```
Comment (jaimergp) [inline docs/source/user-guide/install/linux.rst:9]:
```suggestion
   * `Anaconda Distribution installer for Linux <https://www.anaconda.com/download/>`__.
```

**Review by jaimergp — COMMENTED** [review]
Sorry about the noise with the double underscores, but that's the recommended way of linking to external contents ([using anonymous references](https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use)).

The admonitions about the TOS are really welcome, but it makes me wonder whether this is the place for such information. Maybe the documentation of the open-source project shouldn't be promoting the use of potentially commercial products directly, and only recommend the use of community driven projects.

Maybe it's more of a long-term goal, where we progressively teach the user base about Miniconda vs Miniforge. In that vein, could we preferentially mention Miniforge always first, then Miniconda second? (e.g. in the frontpage, the first tile should be Miniforge?).

**Review by kathatherine — COMMENTED** [review]
I have a few small formatting suggestions. I'm not finding anything in my search that you didn't cover in the main pages where people would most often go. It's not advisable to drop these disclaimers everywhere we mention `defaults` throughout.

I think, Jaime, that while things are a bit in flux, it's important to get the information out there about the TOS. We should do our best not to hide it, even if it is promoting a commercial entity. Once channel additions are explicit all over, we can pull back more on the notes about the TOS and maybe just link to information the Anaconda docs as a secondary option?

**Comment by jezdez** [comment]
> Sorry about the noise with the double underscores, but that's the recommended way of linking to external contents ([using anonymous references](https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use)).

I don't agree with the use of anonymous references and would rather clean up the links, so URL can be reused (and therefor easily changed). Unfortunately, the conda docs aren't really well-written rST in some cases, like link management.

> The admonitions about the TOS are really welcome, but it makes me wonder whether this is the place for such information. Maybe the documentation of the open-source project shouldn't be promoting the use of potentially commercial products directly, and only recommend the use of community driven projects.

There is already information about defaults in the docs and I didn't suggest removing it, but augmenting it with information about conda-forge. Basically, IMO we have to provide a way forward for existing conda users, and we can't just omit this information. There is also no alternative documentation of the various distributions currently to point to. Writing that is out of the scope of the conda 24.9.0 release. That said, I absolutely believe that this should be the end goal of the deprecation process.

Removing the Anaconda Distribution from the docs and replacing it with conda-forge would basically create more concerns "how to get started" with conda, since conda-forge isn't an actual alternative for many users who are looking for a commercially supported product. Since the conda-forge project hasn't indicated wanting to provide the same safeguards as Anaconda Distribution (being a community project), the best way forward is iteratively updating the docs, mentioning them both and eventually replacing that information, once we're further along with the channel registry CEP. I would assume we'd have a place where the various distributions would be compared in detail.

> Maybe it's more of a long-term goal, where we progressively teach the user base about Miniconda vs Miniforge. In that vein, could we preferentially mention Miniforge always first, then Miniconda second? (e.g. in the frontpage, the first tile should be Miniforge?).

I think that's a good next step in the November release, since it requires providing a detailed explanation for why conda-forge is preferred. I did remove the tab interface by the way, to make it visually not secondary, compared to miniconda, which, I hoped, would reduce the frustration of conda-forge users.

File: docs/source/user-guide/getting-started.rst (hunk 9 above)
Comment (jezdez) [reply docs/source/user-guide/getting-started.rst:20]: In this case, it's referring to the conda project, not the `conda` command, [so no need to put it in code formatting](https://github.com/conda/ceps?tab=readme-ov-file#conda-capitalization-standards). That's why it's correct to capitalize it, too.

**Comment by jezdez** [comment]
> It's not advisable to drop these disclaimers everywhere we mention `defaults` throughout.

@kathatherine Just checking if I understood this correctly, would you say I should not mention it in the places that this PR proposes? Should we keep that information in only one of the guides?

**Comment by kathatherine** [comment]
> > It's not advisable to drop these disclaimers everywhere we mention `defaults` throughout.
> 
> @kathatherine Just checking if I understood this correctly, would you say I should not mention it in the places that this PR proposes? Should we keep that information in only one of the guides?

Sorry, @jezdez , I wasn't clear. I think your additions are great. We have lots of other places we mention "defaults" outside of the places you have edited, but I don't think they need the extra context. For instance, https://docs.conda.io/projects/conda/en/stable/user-guide/tasks/manage-channels.html mentions defaults, but points to https://docs.conda.io/projects/conda/en/stable/user-guide/concepts/channels.html, which mentions terms of service limitations.

The initial issue quotes a comment that talks about updating the conda Readme as well. Is that something we also want to do in this PR?

**Comment by jezdez** [comment]
@kathatherine I've updated it in 03e6576ef160f7cac674d0088e6907de481f2b87

**Review by jaimergp — APPROVED** [review]

**Review by kathatherine — APPROVED** [review]
Thanks so much, Jannis!

**Pull request merged**